Vagrantfile

```diff
@@ -0,0 +1,15 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "fedora/32-cloud-base"
+  config.vm.provider "virtualbox" do |v|
+  	v.memory = 2048
+  	v.cpus = 2
+  end
+  config.vm.synced_folder "init_scripts/", "/init_scripts", create: false
+  config.vm.synced_folder "selotape/", "/home/vagrant/selotape", create: true
```

Comment:
The project files are synced automatically to /vagrant.  What is the purpose of syncing those files again?

Reply:
From my usage, the /vagrant folder wasn't syncing properly, reading online this fixes this issue.

Reply:
Take a look at this option: https://developer.fedoraproject.org/tools/vagrant/vagrant-sshfs.html

Reply:
Thanks, this looks good, but its similar to the configuration I've set before, just uses a different method for the file system of the storage (ssh-fs).
Anyway, I'm updating the machine to Ubuntu (as described in one of the other comments), and the default /vagrant folder works fine in there, so we can remove this shared folders in the next push.